### PR TITLE
Feature close project

### PR DESCRIPTION
**Pull request by ajith1994**
Added the 'Close Project' menu item. 
This allows the project to be closed and all the open windows and context sensitive menu associated with the project are closed.

### CODE REVIEW

**Review by gravestench — DISMISSED** [review]
Good work!

File: hsapp/menubar.go [hunk 1]
```diff
@@ -58,6 +58,7 @@ func (a *App) renderMainMenuBar() {
 					}
 				}),
 			}),
+			g.MenuItem("Close Project").OnClick(a.onExitProjectClicked),
```
Comment (gucio321) [inline hsapp/menubar.go]: lets add `Enabled( a.project != nil)`

Comment (ajith1994) [reply hsapp/menubar.go]: Thank you for pointing that out! I made the necessary changes.

File: hsapp/menubar.go [hunk 2]
```diff
@@ -182,6 +184,14 @@ func (a *App) onFilePreferencesClicked() {
 	a.preferencesDialog.Show(a.config)
 }
 
+func (a *App) onExitProjectClicked() {
+	a.project = nil
```
Comment (gucio321) [inline hsapp/menubar.go]: maybe also call `a.project.Save()` here

Comment (ajith1994) [reply hsapp/menubar.go]: I feel that there should be a dialog telling the user: "Do you want to save the project before closing? Yes/No/Cancel"
This way, if the user wanted to get rid of all the changes, that they made during the hellspawner session, they would have the option to do so.

Comment (gucio321) [reply hsapp/menubar.go]: for yes-no dialog, you can use `dialog` package we use in a few other places:
```golang
if save := dialog.Message("Do you want to save current project?").YesNo(); save {
       a.project.Save()
}
```

**Comment by gravestench** [comment]
I've issued a PR against your fork which resolves merge conflicts

**Review by gravestench — DISMISSED** [review]

File: hsapp/menubar.go (hunk 2 above)
Comment (ajith1994) [reply hsapp/menubar.go:188]: I will implement the dialog box in another PR.

**Review by gravestench — APPROVED** [review]
good work!

**Pull request merged**